src/fides/api/ops/db/base_class.py

```diff
@@ -20,7 +19,6 @@
 from fides.api.ops.util.text import to_snake_case
 
 T = TypeVar("T", bound="OrmWrappedFidesBase")
-ALLOWED_CHARS = re.compile(r"[A-z0-9\-_]")
```

Comment:
As @adamsachs said, remnants of the fidesops/fidesctl merge.  An earlier version of this was in solon:

https://github.com/ethyca/solon/commit/1b2aa1bc509338045350185619e78d36975cd52a#diff-e6e83ed3d3f8c6a40f4532ab5b6233b24ff79303fcaa6ae12bb08689e546ed18R220

Reply:
I haven't heard that word in a long time, nostalgic haha